custom_components/versatile_thermostat/base_thermostat.py

```diff
@@ -853,6 +855,22 @@ def swing_modes(self) -> list[str] | None:
         """
         return None
 
+    @property
```

Comment:
Hello, I think this is not necessary in the base class but only in the thermostat_climate.py. This could be safely removed I think.